src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java

```diff
@@ -31,13 +36,13 @@ public class InputFieldAnnotations {
 
   @SomeAnnotation1 @SomeAnnotation2 boolean test = false;
 
-  @SuppressWarnings("all") @SomeAnnotation3(x = 0) float pi = 3.14f;
+  @Deprecated @SomeAnnotation3(x = 0) float pi = 3.14f;
```

Comment:
There is no violation here, can we revert it to `@SuppressWarnings`?

Reply:
I also do not understand this update, please explain.

Reply:
I got it, all suppressions are activated in style guide https://checkstyle.org/google_style.html#Google_Suppressions
https://checkstyle.org/checks/annotation/suppresswarningsholder.html#Description

. It is possible to suppress all the checkstyle warnings with the argument "all".

please make annotation like @SuppressWarnings("bla")

Reply:
done